app/gui/src/presenter/graph.rs

```diff
@@ -82,6 +84,7 @@ struct Model {
     view:             view::graph_editor::GraphEditor,
     state:            Rc<State>,
     _visualization:   Visualization,
+    widgets:          Widgets,
```

Comment:
are these widgets or a widget controller / multiplexer? I believe these are not real widgets. So the var name is confusing.